doc/quickstart.rst

```diff
@@ -0,0 +1,40 @@
+##########
+Quickstart
+##########
+
+Welcome to our Quickstart!
+Here you will find an interactive tour that will guide you through Spyder's interface. 
+You will get familiar with the most important parts of the IDE, especially those we will be mentioning throughout our docs. 
+Finally, you will get to walk through some of Spyder's key panes and functionality.
```

Comment:
```suggestion
You'll get familiar with the most important parts of the IDE, especially those we'll be mentioning throughout our docs. 
Finally, you'll get to walk through some of Spyder's key panes and functionality.
```

Too many `will`s make things too formal, in my opinion.

Reply:
That's how I would write it for a native English speaker, but per the style guide and our previous discussions and PRs, contractions should be strongly avoided due to the increase in difficulty for non-native speakers. So I'm not sure what we should do here.

Reply:
I think in this case is fine to use contractions to make text a bit friendlier. But let's see @juanis2112 thinks.